### ISSUE TITLE
build with OPENMP

### ISSUE THREAD

**Issue by KrisThielemans** [conda-forge/stir-feedstock#8]
Currently, STIR still doesn't default to using OPENMP. We could simply switch `STIR_OPENMP=ON`. Does this need anything special? e.g. dependency on https://anaconda.org/conda-forge/openmp? (which confusingly seems to be for llvm compilers only)

**Comment by paskino** [comment]
For what regards OSX and conda, I [use ](https://github.com/vais-ral/CCPi-Framework/blob/fecff8ded735d309aba43d30226c0bb51386c905/recipe/meta.yaml)`openmp` conda-forge package in the `build` and `run` sections. 

For OSX, the `cmake` command requires you to add the additional flags `OPENMP_INCLUDES` and `OPENMP_LIBRARIES` to the appropiate locations as in [here](https://github.com/vais-ral/CCPi-Framework/blob/fecff8ded735d309aba43d30226c0bb51386c905/recipe/build.sh#L21L27).

For other systems, I suppose cmake is able to find appropriate libraries by itself and (apparently) they are available on **most** OS variants.

**Comment by KrisThielemans** [comment]
I tried this on ubuntu 18.04. `conda install stir` doesn't get me the 3rd openmp build, but sticks to the previous one.No idea why, but I guess i'll try to remove the openmp dependency. @paskino @casperdcl  any better ideas?

**Comment by KrisThielemans** [comment]
Fixed with #15

**Comment by KrisThielemans** [comment]
we might have run-time errors related to libgomp, see this warning in #17  https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=146210&view=logs&j=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e&t=6d4b912b-175d-51da-0fd9-4d30fe1eb4e7&l=7522
```
WARNING (stir,bin/correct_projdata): Needed DSO lib/libgomp.so.1 found in ['_openmp_mutex']
WARNING (stir,bin/correct_projdata): .. but ['_openmp_mutex'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
```
